packages/fxa-shared/db/mysql.ts

```diff
@@ -251,6 +259,18 @@ export class MysqlStoreShared {
           return reject(err);
         }
 
+        // The mysql module's connection pool never closes any of its
+        // connections.  There's at least one open contributor PR on the
+        // module's repo for the feature, but not much activity.
+        if (conn._fxa_idle_timeout) {
+          clearTimeout(conn._fxa_idle_timeout);
+        }
+
+        conn._fxa_idle_timeout = setTimeout(() => {
+          (pool as any)._purgeConnection(conn);
```

Comment:
There are cases where the call to _purgeConnection can throw an error. Since this is pretty deep in the code, and happens in a setTimeout, should there be some error handling?

Reply:
A fine idea.  How do you feel about sending a StatsD stat and writing the error to the log?  There isn't any recovery we can do in here.

Reply:
A statsd call would be great.